### PR TITLE
Updating inputs for MABSelector to new types LearningPolicyType and NeighboorhoodPolicyType from the new version of mabwiser

### PR DESCRIPTION
**Pull request by AshishPvjs**

### CODE REVIEW

**Comment by AshishPvjs** [comment]
@skadio

**Review by N-Wouda — APPROVED** [review]
Hi @AshishPvjs, thanks for the PR! Looks good to me, except for the one issue with the types. I'll commit that and then merge this in.

File: pyproject.toml [hunk 1]
```diff
@@ -33,7 +33,7 @@ classifiers = [
 python = "^3.8, <4.0"
 numpy = ">=1.18.0"
 matplotlib = ">=3.5.0"
-mabwiser = { version = ">=2.7.0", optional = true }
+mabwiser = { version = ">=2.7.1", optional = true }
```
Comment (skadio) [inline pyproject.toml:36]: @N-Wouda  just checking if we need to update the ALNS version somewhere --and I don't know how the PyPI thing works for ALNS to reflect the change

**Review by skadio — APPROVED** [review]
blank approve --

File: alns/select/MABSelector.py [hunk 2]
```diff
@@ -79,8 +79,8 @@ def __init__(
         scores: List[float],
         num_destroy: int,
         num_repair: int,
-        learning_policy: "LearningPolicy",
-        neighborhood_policy: Optional["NeighborhoodPolicy"] = None,
+        learning_policy: LearningPolicyType,
+        neighborhood_policy: Optional[NeighborhoodPolicyType] = None,
```
[the remark this answers is not in the document]
Comment (N-Wouda) [reply alns/select/MABSelector.py]: Added that to my TODO list with #163!

**Comment by AshishPvjs** [comment]
Thank you @N-Wouda.

**Comment by skadio** [comment]
thank you as well @AshishPvjs  --also for making the change in MABWiser side

File: pyproject.toml (hunk 1 above)
Comment (N-Wouda) [reply pyproject.toml:36]: I'll bump versions now and release to PyPI, no worries!

**Comment by N-Wouda** [comment]
New release is underway: https://github.com/N-Wouda/ALNS/actions/runs/5755818928. Should hit PyPI in a few minutes! 🚀

**Comment by N-Wouda** [comment]
`alns==5.3.1` now requires `mabwiser >= 2.7.1`!

**Pull request merged**